dist/react-select.js

```diff
@@ -1110,4 +1112,4 @@ module.exports = Value;
 
 }).call(this,typeof global !== "undefined" ? global : typeof self !== "undefined" ? self : typeof window !== "undefined" ? window : {})
 },{}]},{},[2])(2)
-});
\ No newline at end of file
+});
```

Comment:
check for the changes in this line.

Reply:
An extra new line got appended while generating build.